ansible/roles/aws_docker_setup/tasks/minimal.yml

```diff
@@ -1,48 +1,51 @@
 ---
-  - name: Created docker group
-    group:
-      name: docker
-      state: present
+  - name: Configure docker
+    become: true
+    block:
+    - name: Created docker group
+      group:
+        name: docker
+        state: present
 
-  - name: Added ec2-user to group
-    user:
-      name: ec2-user
-      groups: "ec2-user,adm,wheel,systemd-journal,docker"
+    - name: Added ec2-user to group
```

Comment:
Just checking if this formatting change is intentional.

Reply:
Do you mean the tabbing out? I added several "become: true" blocks to replace a play-wide "become: true" that used to exist on the AWS setup tasks. Settings "become: true" for the entire play wasn't working nicely with the k8s automation.